src/stores/rootStore.ts

```diff
@@ -116,6 +118,16 @@ export const useRootStore = defineStore('root', {
       return (state.entityStatus === EntityStatus.FILED_APP)
     },
 
+    /** Is True if this is on local filing pages. */
+    isInFilingPage (state: StateIF): boolean {
+      return state.currentPage === Routes.ANNUAL_REPORT ||
```

Comment:
Could you just check the route here, without needing to save it?

Reply:
If you end up not needing to store the route name (current page) in the store, and since this getter is only used in one place, consider using a computed (local getter) in EntityInfo.vue -- or possibly EntityMenu.vue itself.

Reply:
Yeah, it's more straightforward to use local getter. I've updated it.